chainer/functions/math/sum.py

```diff
@@ -6,7 +6,9 @@
 class Sum(function.Function):
     """Sum of array elements over a given axis."""
 
-    def __init__(self, axis=None):
+    keepdims = False
```

Comment:
This is for pickled files. When a user unpickle an object, `__init__` method is not called. If an object is pickled with the old implementation, it causes an error.